### DISCUSSION TITLE
Do you know a LSP Server for PostCSS?

### DISCUSSION THREAD

**Discussion by stephane-klein** [postcss/postcss#1769] [Q&A]
Hi,

Do you know a [LSP Server](https://en.wikipedia.org/wiki/Language_Server_Protocol) for `PostCSS` language?

I looked https://github.com/microsoft/vscode-css-languageservice but I don't feel that this project support PostCSS language (See this issue https://github.com/microsoft/vscode-css-languageservice/issues/283).

Best regards,  
Stéphane

**Comment by ai** [comment]
I never heard about it.
